docker/scripts/install_ros_deps.sh

```diff
@@ -0,0 +1,33 @@
+# Set up a ROS workspace
+mkdir -p $MOMA_DEP_WS/src
+cd $MOMA_DEP_WS
+catkin init
+catkin config --extend /opt/ros/noetic
+catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release
+
+# Install ROS packages from apt
+apt-get install -y \
```

Comment:
Is this script always called after another script that already calls apt update? Otherwise you might want to prepend it with `apt-get update &&` to make sure apt's lists are available.